AK/Time.h

```diff
@@ -139,7 +139,7 @@ class Time {
     }
     [[nodiscard]] static Time from_timespec(const struct timespec&);
     [[nodiscard]] static Time from_timeval(const struct timeval&);
-    [[nodiscard]] constexpr static Time min() { return Time(-0x8000'0000'0000'0000LL, 0); };
+    [[nodiscard]] constexpr static Time min() { return Time(-__INT64_MAX__ - 1LL, 0); };
     [[nodiscard]] constexpr static Time zero() { return Time(0, 0); };
     [[nodiscard]] constexpr static Time max() { return Time(0x7fff'ffff'ffff'ffffLL, 999'999'999); };
```

Comment:
If you insist on using INT64_MAX/MIN, then you should use it here, too.

Reply:
good point, symmetry would be nice here.